Directory.Build.props

```diff
@@ -26,6 +26,9 @@
     <TargetFramework>$(URHO3D_NETFX)</TargetFramework>
     <RuntimeIdentifier>$(URHO3D_NETFX_RUNTIME_IDENTIFIER)</RuntimeIdentifier>
   </PropertyGroup>
+  <PropertyGroup Condition="$([System.Text.RegularExpressions.Regex]::IsMatch($(URHO3D_NETFX), '^net(46|standard).*$'))">
```

Comment:
Instead of doing same thing in two places (cmake and props) you should add `URHO3D_NETFX_LEGACY_VERSION` to `CMake/CMake.props.in` file and use it when appending `DefineConstants`.